css/stanford_jumpstart_engineering.css

```diff
@@ -51,4 +51,37 @@
   .style-custom-styles-rich #block-bean-jse-linked-logo-block p a {
     padding-bottom: 10px;
   }
-}
\ No newline at end of file
+}
+
+/* grouped grid layout for people */
+
+.views-grid-four .views-row.views-row-4,
+.views-grid-four .views-row.views-row-8,
+.views-grid-four .views-row.views-row-12,
+.views-grid-four .views-row.views-row-16,
+.views-grid-four .views-row.views-row-20,
+.views-grid-four .views-row.views-row-24,
+.views-grid-four .views-row.views-row-28,
+.views-grid-four .views-row.views-row-32,
+.views-grid-four .views-row.views-row-36,
+.views-grid-four .views-row.views-row-40,
+.views-grid-four .views-row.views-row-44,
+.views-grid-four .views-row.views-row-48,
+.views-grid-four .views-row.views-row-52,
+.views-grid-four .views-row.views-row-56,
+.views-grid-four .views-row.views-row-60,
+.views-grid-four .views-row.views-row-64,
+.views-grid-four .views-row.views-row-68,
+.views-grid-four .views-row.views-row-72,
+.views-grid-four .views-row.views-row-76,
+.views-grid-four .views-row.views-row-80,
+.views-grid-four .views-row.views-row-84,
+.views-grid-four .views-row.views-row-88,
+.views-grid-four .views-row.views-row-92,
+.views-grid-four .views-row.views-row-96,
+.views-grid-four .views-row.views-row-100,
+.views-grid-four .views-row.views-row-104,
+.views-grid-four .views-row.views-row-108,
+.views-grid-four .views-row.views-row-112 {
```

Comment:
When the 116th item is added to a grouping, the layout would flow improperly (previously, this was only styled for 28 items). We can make this as big as needed.

I'm confident that there is a way to address this with the adjacent and/or nth-child CSS selectors. After trying several equations, however, I was unable to get this reduced down to a few lines of code (as the view's groupings breaks the standard nth-child convention - I tried to target every fourth item, but it recalculates with each grouping).

I filed a big in OFW, which is where this code lives. Brian poked at it for a bit, too. No luck... yet. If and when we have a more succinct fix, I'll add it to OFW and SOE.

Due to the timeliness of this, I thought adding the above fix was the best route to take now.

Let me know your thoughts.